rocAL/rocAL/source/parameters/parameter_crop.cpp

```diff
@@ -57,8 +57,6 @@ void CropParam::array_init()
     croph_arr_val.resize(batch_size);
     x2_arr_val.resize(batch_size);
     y2_arr_val.resize(batch_size);
-    in_width.resize(batch_size);
-    in_height.resize(batch_size);
```

Comment:
Please check if this deletion is intended

Reply:
Yes its intended